Apps/APN/babel.config.js

```diff
@@ -1,7 +1,3 @@
-module.exports = function (api) {
-  api.cache(true);
```

Comment:
removing this was on purpose?

Reply:
Yes, this is a part of the [version upgrade](https://react-native-community.github.io/upgrade-helper/?from=0.72.2&to=0.74.4#RnDiffApp-babel.config.js).